tests/test_exporter.py

```diff
@@ -1130,6 +1130,43 @@ def test_exporter_can_export_requirements_txt_with_nested_packages_cyclic(
     assert content == expected
 
 
+def test_exporter_can_export_requirements_txt_with_circular_dependency(
```

Comment:
maybe

```suggestion
def test_exporter_can_export_requirements_txt_with_circular_root_dependency(
```

At the moment, it's not clear from its name that the focus of the test is a circular dependency _on the root package_.